frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.ts

```diff
@@ -0,0 +1,207 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { BaseComponent } from '@core/base/base.component';
+import { getDisplayDate } from '@core/helpers/date-helper';
+import { getDisplayDuration } from '@core/helpers/display-duration-hepler';
+import { IDuration } from '@core/models/IDuration';
+import { INewMeeting } from '@core/models/INewMeeting';
+import { INewMeetingMember } from '@core/models/INewMeetingTeamMember';
+import { NewMeetingService } from '@core/services/new-meeting.service';
+import { NotificationService } from '@core/services/notification.service';
+import { naturalNumberRegex, newMeetingNameRegex } from '@shared/constants/model-validation';
+import { LocationType } from '@shared/enums/locationType';
+import { UnitOfTime } from '@shared/enums/unitOfTime';
+import { map, Observable, startWith } from 'rxjs';
+
+@Component({
+    selector: 'app-new-meeting',
+    templateUrl: './new-meeting.component.html',
+    styleUrls: ['./new-meeting.component.sass'],
+})
+export class NewMeetingComponent extends BaseComponent implements OnInit {
+    constructor(private newMeetingService: NewMeetingService, public notificationService: NotificationService) {
+        super();
+        this.getTeamMembersOfCurrentUser();
+    }
+
+    public teamMembers: INewMeetingMember[];
+
+    public addedMembers: INewMeetingMember[] = [];
+
+    public filteredOptions: Observable<INewMeetingMember[]>;
+
+    public durations: IDuration[] = getDisplayDuration();
+
+    public dates = getDisplayDate();
+
+    public locations = Object.keys(LocationType).filter(key => Number.isNaN(Number(key)));
+
+    public unitOfTime = Object.keys(UnitOfTime).filter(key => Number.isNaN(Number(key)));
```

Comment:
Why do we need a filter here?

Reply:
I need a filter to display enums properly in element
Before using filter:
![image](https://user-images.githubusercontent.com/98499085/187046012-06bd2b4a-2823-46d2-955e-0aaf25d9aa37.png)
After using filter:
![image](https://user-images.githubusercontent.com/98499085/187046035-a0450dfc-7652-428e-b059-61e998fe0b8c.png)

Reply:
This is the problem with using enums
We don't need filter here

Reply:
Refactored it.